hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/configuration/DeltaConfig.java

```diff
@@ -189,6 +190,10 @@ public boolean validateClean() {
       return Boolean.valueOf(configsMap.getOrDefault(VALIDATE_CLEAN, false).toString());
     }
 
+    public boolean doEnableRowWriting() {
```

Comment:
doEnableRowWriting -> doRowWriting or enableRowWriting

Reply:
I usually get a confusion if we can name the method same as variable name and hence didn't name the method "enableRowWriting". do you know whats the guideline here